airbyte-integrations/connectors/source-wrike/sample_files/configured_catalog.json

```diff
@@ -0,0 +1,133 @@
+{
+  "streams": [
+    {
+      "stream": {
+        "name": "tasks",
+        "json_schema": {
```

Comment:
you can delete this incase you are already using the one in integration_tests folder